server/testserver.go

```diff
@@ -28,8 +28,19 @@ import (
 	"github.com/cockroachdb/cockroach/storage/engine"
 	"github.com/cockroachdb/cockroach/util"
 	"github.com/cockroachdb/cockroach/util/hlc"
+	"github.com/cockroachdb/cockroach/util/log"
 )
 
+// StartTestServer starts a in-memory test server.
+func StartTestServer() *TestServer {
```

Comment:
It used to, but we also use StartTestServer from examples. Any error in s.Start() will cause a nil pointer panic rather than a nicely-formatted error message.